spec/javascripts/member-ui/fundraiser_spec.js

```diff
@@ -260,7 +260,7 @@ describe("Fundraiser", function() {
       }
 
       beforeEach(function(){
-        suite.followUpUrl = "/pages/636/follow-up";
+        suite.followUpUrl = `/api/member_authentication/new?page_id=1&email=`;
```

Comment:
You're going to have to bring fundraiser_spec.js up to date - you've got failures if you fix this syntax error. It would also be great to get this updated so it test for critical parts of the new logic you added.